### PR TITLE
More updates

### PR DESCRIPTION
**Pull request by anatheka**
Updats and additions for comments from Clouditor branch (https://github.com/clouditor/clouditor/pull/1305):
- Add DataHasValue
- Update proto types
- Ignore object and data properties with "prop:parent" 
- Use label names instead of IRIs
- Start message comment with "X is an entity in our Cloud ontology"
- Add proto type map  for "xsd:java.util.Map<String, String>"

### CODE REVIEW

File: proto.go [hunk 1]
```diff
@@ -1,4 +1,3 @@
 package owl2protobuf
```
Comment (oxisto) [inline proto.go]: I think we can delete this file alltogether?

File: internal/util/util.go [hunk 2]
```diff
@@ -59,21 +60,29 @@ func isResourceAboveX(resource *ontology.Resource, preparedOntology ontology.Ont
...
-	case "xsd:java.time.Duration":
-		return "time.duration"
+		return "float"
+	case "xsd:java.time.Duration", "xsd:dateTime":
+		return "int64"
+	case "xsd:java.time.ZonedDateTime":
+		return "google.protobuf.Timestamp"
+	case "xsd:java.util.ArrayList<Short>":
+		return "repeated uint32"
+	case "xsd:java.util.Map<String, String>": // TODO(oxisto): Do we want to use here maps, as in the CPG?
```
Comment (oxisto) [inline internal/util/util.go:86]: yes, these need to be maps

File: proto.go (hunk 1 above)
Comment (anatheka) [reply proto.go]: If we do not want to generate protobuf files only by this tool, yes.

File: internal/util/util.go (hunk 2 above)
Comment (anatheka) [reply internal/util/util.go:86]: Done.

**Comment by anatheka** [comment]
We should rename the tool to owl2proto.

**Comment by oxisto** [comment]
> We should rename the tool to owl2proto.

Good idea

File: cmd/owl2protobuf/owl2protobuf.go [hunk 3]
```diff
@@ -135,12 +195,16 @@ message ResourceID {
 
 	// Create proto messages with comments
 	for _, v := range preparedOntology.Resources {
+		if v.Iri == "http://graph.clouditor.io/classes/ApplicationLog" {
```
Comment (oxisto) [inline cmd/owl2protobuf/owl2protobuf.go]: Is this needed? Maybe configurable?

Comment (anatheka) [reply cmd/owl2protobuf/owl2protobuf.go]: :) that was for debugging reasons.

**Review by oxisto — APPROVED** [review]

**Pull request merged**